__tests__/business-text/business-text.query.test.js

```diff
@@ -0,0 +1,304 @@
+            value: 'НоваБТ',
+          },
+          {
+            __typename: 'Language',
+            lang: 'en',
+            value: 'NewBT',
+          },
+        ]);
+      expect(businessText.text).toBeInstanceOf(Array);
+      expect(businessText)
+        .toHaveProperty('text', [
+          {
+            __typename: 'Language',
+            lang: 'uk',
+            value: 'Тут бізнес текст',
+          },
+          {
+            __typename: 'Language',
+            lang: 'en',
+            value: 'Business text here',
+          },
+        ]);
+    } catch (e) {
+      console.error(e);
+    }
+  });
+
+  test('#3 Returning not existing business text should return error message', async () => {
+    const res = await client
+      .query({
+        query: gql`
+          query($id: ID!) {
+              getBusinessTextById(id: $id) {
+                  ... on BusinessText {
+                      code
+                      title {
+                          value
+                          lang
+                      }
+                      text {
+                          lang
+                          value
+                      }
+                  }
+                  ... on Error {
+                      statusCode
+                      message
+                  }
+              }
+          }
+      `,
+        variables: { id: notExistBusinessTextId },
+      })
+      .catch(e => e);
+    expect(res.data.getBusinessTextById).toBeDefined();
+    expect(res.data.getBusinessTextById).toHaveProperty('statusCode', 404);
+    expect(res.data.getBusinessTextById).toHaveProperty(
+      'message',
+      BUSINESS_TEXT_NOT_FOUND,
+    );
+  });
+
+  test('#4 Should receive selected business text by code', async () => {
+    try {
+      const res = await client
+        .query({
+          query: gql`
+            query($code: String!) {
+                getBusinessTextByCode(code: $code) {
+                    ... on BusinessText {
+                        code
+                        title {
+                            value
+                            lang
+                        }
+                        text {
+                            lang
+                            value
+                        }
+                    }
+                    ... on Error {
+                        statusCode
+                        message
+                    }
+                }
+            }
+        `,
+          variables: { code: 'new-code' },
+        })
+        .catch(e => e);
+
+      businessText = res.data.getBusinessTextByCode
+
+      expect(businessText).toBeDefined();
+      expect(businessText).toHaveProperty(
+        'code',
+        'new-code',
+      );
+      expect(businessText.title)
+        .toBeInstanceOf(Array);
+      expect(businessText)
+        .toHaveProperty('title', [
+          {
+            __typename: 'Language',
+            lang: 'uk',
+            value: 'НоваБТ',
```

Comment:
Please, use here values from the `newBusinessText` instead of plain hardcode literals. It would be much easier and safe to changes some values in `newBusinessText` and be sure that it is changes in tests too. Use hardcode values only for properties that you cannot take from `newBusinessText`